### DISCUSSION TITLE
facetimehd camera on macbook air

### DISCUSSION THREAD

**Discussion by stardustdelux** [sebanc/brunch#1072] [Q&A]
Hi,
First of all many thanks for your effort for bringing Chrome OS to "non-Chromebooks" computers.
I have a Macbook Air and installed Chrome OS. Now I am wondering whether it is possible to get the facetimehd camera working? Is there a similar installation procedure as for linux (bcwc_pcie)?
Many thanks
Marc

**Comment by LillyYaGirl** [comment]
I had the same problem with my 2012 macbook pro. open the shell (control + alt + T), type "shell" and hit enter, then type "sudo edit-grub-config" and it will let you edit the grub config file, where you want to type "no_camera_config" right after "options=" (NO SPACES)

**Comment by tejasraman** [comment]
You could try this:

[https://gist.github.com/ukn/a2f85e3420ae7d0f64db2274a9bc106b](https://gist.github.com/ukn/a2f85e3420ae7d0f64db2274a9bc106b)

Or this

[https://github.com/patjak/facetimehd/wiki](https://github.com/patjak/facetimehd/wiki)

**Reply by weberk** [reply]
No chance. ChromeOS does only handle USB cams. 
In my mid 2013 MacBook Air the cam was connected to PCI bus.

**Comment by tejasraman** [comment]
It was worth a shot :)
